Helper/Api/Orders.php

```diff
@@ -618,4 +598,143 @@ public function getOrders($storeId, $page, $pageSize): OrderSearchResultInterfac
 
         return $this->orderRepository->getList($searchCriteria);
     }
+
+    /**
+     * @param $item
+     * @return float
+     */
+    private function getItemPrice($item): float
+    {
+        $itemPrice = 0.0;
+        $parentProduct = $item->getParentItem();
+        if ($parentProduct) {
+            $parentProductType = $parentProduct->getProductType();
+            if ($parentProductType == 'bundle') {
+                $itemPrice = $item->getPrice();
+            } elseif ($parentProductType == 'configurable') {
+                $itemPrice = $parentProduct->getPrice();
+            }
+        } else {
+            $itemPrice = $item->getPrice();
+        }
+
+        return $itemPrice;
+    }
+
+    /**
+     * @param $item
+     * @param $taxFreeId
+     * @return array
+     */
+    private function getBundlePayload($item, $taxFreeId)
+    {
+        $bundleItemsPriceTotal = null;
+        $bundlePrice = $item->getPriceInclTax();
+
+        $parentProduct = $this->getParentProductData($item);
+
+        foreach ($item->getChildrenItems() as $bundleItem) {
+            $bundleItemSku = $bundleItem->getSku();
+            $bundleItemPrice = $bundleItem->getProduct()->getPrice();
+            $bundleItemsPriceTotal += $bundleItemPrice;
```

Comment:
we should use something like https://github.com/brick/money for money calculation, otherwise, there will be calculation errors. Float is not a good way to represent monetary values.